.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
markdown has to have trailing whitespace - see below
```
[*.md]
trim_trailing_whitespace = false
```

Reply:
Added to .editorconfig